content/assets/stylesheets/includes/about.scss

```diff
@@ -70,6 +70,9 @@ table.board-table {
   justify-content: center;
   align-items: center;
 
+  margin-left: 0;
+  margin-top: 0;
```

Comment:
The circles were out of balance due to using a `ul` element inside of content. This is just a good enough fix, a better one would be to use bulma's columns system